tests/publiccloud/prepare_tools.pm

```diff
@@ -122,6 +122,30 @@ EOT
     validate_script_output("terraform -version", qr/$terraform_version/);
     record_info('Terraform', script_output('terraform -version'));
 
+    # Ansible install with pip
+    # Default version is chosen as low as possible so it run also on SLE12's
+    # ANSIBLE_CORE_VERSION should be set only if the different then default one need to be used
+    my $ansible_version = get_var('ANSIBLE_VERSION', '4.10.0');
+    my $ansible_core_version = get_var('ANSIBLE_CORE_VERSION');
+    my $ansible_install_log = '/tmp/ansible_install.log';
+
+    if ( script_run("python3.9 -m pip -V") == 0 ) {
```

Comment:
Since `python39-pip` is installed and we use it for other modules as well, I think this check is not necessary. We can safely assume pip is present.

Reply:
I was think same so OK I will remove it.